### DISCUSSION TITLE
Add new configuration files for Mask2Former VIS algorithm in mmdetection. 给 mmdet Mask2Former VIS 算法增加新配置文件

### DISCUSSION THREAD

**Discussion by OpenMMLab-Assistant-004** [open-mmlab/OpenMMLabCamp#487] [MMDetection]
# Add new configuration files for Mask2Former VIS algorithm in mmdetection. 
 给 mmdet Mask2Former VIS 算法增加新配置文件 

According to the [Config adaptation document]([https://aicarrier.feishu.cn/docx/PkICdaJbpoOVjLxMFi2cYqPjnOc)](https://aicarrier.feishu.cn/docx/PkICdaJbpoOVjLxMFi2cYqPjnOc))  and 
 the supported algorithm configurations, add all configurations 
 for Mask2Former VIS and verify the correctness of inference and training.

根据 [Config适配文档]([https://aicarrier.feishu.cn/docx/PkICdaJbpoOVjLxMFi2cYqPjnOc)](https://aicarrier.feishu.cn/docx/PkICdaJbpoOVjLxMFi2cYqPjnOc))  内容和已经支持的算法配置，给 Mask2Former VIS 所有配置
 新增配置并验证推理和训练正确性

## Task Information

**Repository**：MMDetection

**Difficulty**：Easy

**Points**：20

**Estimated Time Consumption**：21 days

## Task Status

**Claimant's Github ID**：Crescent-Saturn

**Claim Time**：2023年10月31日

**PR Link**：https://github.com/open-mmlab/mmdetection/pull/11135

## Points to Note

To claim the task, please reply `Claim the mission` in the comment section. After claiming the task, please add our assistant bot called MeowMewo on WeChat at: `openmmlabwx`.

After completing the task, please submit a PR with the title starting with [MMSIG]. After submission, please reply with your PR link in the comments section of this Discussion. We will review your work as soon as possible.

在评论区回复 `Claim the mission` 领取任务，领取任务后请及时添加小助手喵喵微信: `openmmlabwx`

完成任务后提交相关链接（如是PR，标题请以[MMSIG]开头）回复到本Disucssion评论区，我们会第一时间审核您的贡献。

**Comment by Crescent-Saturn** [comment]
Claim the mission

**Reply by Crescent-Saturn** [reply]
PR link here: [https://github.com/open-mmlab/mmdetection/pull/11135](https://github.com/open-mmlab/mmdetection/pull/11135)
